### PR TITLE
Nt/ios notification fix

### PR DESCRIPTION
**Pull request by noumantahir**
### What does this PR?
Added a workaround to read initial and paused notifications on iOS, retained android side of things to what it was perviously working on.

This was literally the most stupid kind of bug I have encountered in a very long time. Still not able to resolve the source but used a third party package notifee to fix things for us on iOS.

### Issue number
https://github.com/orgs/ecency/projects/2#card-86675629

### Screenshots/Video

https://user-images.githubusercontent.com/6298342/201910280-f264b0eb-b91f-4e76-a8a2-f62eaa8587b8.mov

### CODE REVIEW

**Comment by feruzm** [comment]
@noumantahir there is conflict, could you check?

File: ios-notify-comment.apns [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (feruzm) [inline ios-notify-comment.apns:1]: this can be deleted as well

File: src/screens/application/hook/useInitApplication.tsx [hunk 2]
```diff
@@ -66,13 +78,148 @@ export const useInitApplication = () => {
...
+
+        case 'transfer':
+          routeName = ROUTES.TABBAR.PROFILE;
+          params = {
+            activePage: 2,
+          };
+          break;
+
+        case 'inactive':
+          routeName = ROUTES.SCREENS.EDITOR;
```
Comment (feruzm) [inline src/screens/application/hook/useInitApplication.tsx:204]: favorite and bookmark notifications need route path as well to navigate relevant page

File: ios-notify-comment.apns (hunk 1 above)
Comment (noumantahir) [reply ios-notify-comment.apns]: done

**Review by feruzm — APPROVED** [review]
Lgtm

**Pull request merged**